app/models/ability.rb

```diff
@@ -126,9 +126,10 @@ def owner_perms
       if v_obj
         teams = v_obj.get_team if v_obj.respond_to?(:get_team)
```

Comment:
would be great to have all models response to `:team` instead of some responding to `:team` and some responding to `:get_team`

Reply:
now all respond to :team and implemented `get_teams` for `Media` module as media can exist in multiple teams (used project_medias relation)